lib/mobility-core/src/Kernel/External/Payment/Juspay/Types/Payout.hs

```diff
@@ -0,0 +1,251 @@
+{-
+  Copyright 2022-23, Juspay India Pvt Ltd
+
+  This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License
+
+  as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. This program is
+
+  distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+
+  FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details. You should have received a copy of the GNU Affero
+
+  General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+-}
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE TemplateHaskell #-}
+
+module Kernel.External.Payment.Juspay.Types.Payout where
+
+import Data.Aeson as A
+import Kernel.Beam.Lib.UtilsTH (mkBeamInstancesForEnum)
+import Kernel.Prelude
+
+-- Verify VPA To be Used In future
+
+-- data VerifyVPAReq = VerifyVPAReq
+--   { vpa :: Text,
+--     merchant_id :: Text,
+--     customer_id :: Text,
+--     order_id :: Text
+--   }
+--   deriving stock (Show, Generic)
+--   deriving anyclass (FromJSON, ToJSON)
+
+-- newtype MandateDetails = MandateDetails
+--   {is_handle_supported :: Bool}
+--   deriving stock (Show, Generic)
+--   deriving anyclass (FromJSON, ToJSON)
+
+-- data VPAStatus = VALID | INVALID
+--   deriving (Eq, Show, Read, Generic, ToJSON, FromJSON, ToSchema)
+
+-- $(mkBeamInstancesForEnum ''VPAStatus)
+
+-- data VerifyVPAResp = VerifyVPAResp -- handle error, 400 invalid input data
+--   { vpa :: Text,
+--     status :: VPAStatus,
+--     mandate_details :: MandateDetails,
+--     customer_name :: Text
+--   }
+--   deriving stock (Show, Generic)
+--   deriving anyclass (FromJSON, ToJSON)
+
+--------------------------------------------------------------------------
+-- Payout Create Order Types
+
+data PayoutOrderStatus = READY_FOR_FULFILLMENT | FULFILLMENTS_SCHEDULED | FULFILLMENTS_FAILURE | FULFILLMENTS_SUCCESSFUL | FULFILLMENTS_CANCELLED | FULFILLMENTS_MANUAL_REVIEW | FULFILLED_PARTIALLY | INITIATED | FAILURE | SUCCESS | DISCARDED | MANUAL_REVIEW | CANCELLED | GATEWAY_SWITCHED | ERROR | INVALID | VALID | CONFLICTED | REVERSED
```

Comment:
Write like this

data PayoutOrderStatus 
 = READY_FOR_FULFILLMENT 
 | FULFILLMENTS_SCHEDULED
 | FULFILLMENTS_FAILURE